Microsoft.DotNet.Cli.sln

```diff
@@ -1,7 +1,7 @@
 ﻿
 Microsoft Visual Studio Solution File, Format Version 12.00
-# Visual Studio Version 16
-VisualStudioVersion = 16.0.28515.72
+# Visual Studio 15
```

Comment:
Given that VS 16 is out the door, should we stick it to it? Not really keen on having this file continuously flipped depending on who is editing it.

Reply:
Done